cpp/ql/src/Security/CWE/CWE-416/IteratorToExpiredContainer.qhelp

```diff
@@ -30,6 +30,12 @@ This is because the temporary container is not bound to a rvalue reference.
 </p>
 <sample src="IteratorToExpiredContainerExtendedLifetime.cpp" />
 
+<p>
+To fix <code>lifetime_of_temp_not_extended</code> consider rewriting the code so that the temporary's lifetime is extended.
```

Comment:
Let's remove the apostrophy to make this more understandable by users whose English is not the main language.
```suggestion
To fix <code>lifetime_of_temp_not_extended</code>, consider rewriting the code so that the lifetime of the temporary object is extended.
```